### DISCUSSION TITLE
GitHub Actions: Time from Skipped Jobs

### DISCUSSION THREAD

**Discussion by chingc** [community/community#26456] [Actions]
<p>I have some workflows that will only run jobs if the pull request is not a draft.</p>
<p>These workflows will still run for a few seconds before it exits with a “skipped” status.</p>
<p>Does seconds from “skipped” jobs count against the monthly included minutes?</p>

**Comment by mscoutermarsh** [comment, marked as answer]
<p>No, those workflows never start up a runner, so they do not count towards the included minutes.</p>
<p>Note: the <code>if</code> has to be at the job level to stop the job from starting.</p>
